### ISSUE TITLE
Improve support for Sphinx LaTeX builder

### ISSUE THREAD

**Issue by felixfontein** [ansible-community/antsibull-docs#196]
This issue is tracking what's needed to improve support for the Sphinx LaTeX builder:

- [x] Force long tables, improve column distribution: https://github.com/ansible-community/antsibull-docs/pull/195
- [x] Make nested options/return values work: https://github.com/ansible-community/antsibull-docs/pull/198
- [x] Make collection links section work: https://github.com/ansible-community/antsibull-docs/pull/200
- [x] Improve styling of all the roles used by antsibull-docs: https://github.com/ansible-community/antsibull-docs/pull/199

Ref: https://github.com/ansible/ansible-documentation/issues/109#issuecomment-1684914779

**Comment by felixfontein** [comment]
I've looked into creating the collection links in a way that they can also be rendered in LaTeX. Turns out that isn't so simple; namely adding the `aria-role="button" target="_blank" rel="noopener external"` part is hard. I guess I have to create my own node type (resp. inherit from an existing one), which is doable but brings up potential new problems, like not working with alternate Sphinx builders like rinoh.
